storage-service/src/main/java/io/quarkus/qe/consumers/UpdateRepositoryRequestConsumer.java

```diff
@@ -37,7 +41,7 @@ public void addRepository(Repository repository) {
         LOG.info("Update repository " + repository.getRepoUrl());
 
         RepositoryEntity entity = RepositoryEntity.findById(repository.getId());
-        entity.name = repository.getName();
+        entity.name = Optional.ofNullable(repository.getName()).orElse(UNDEFINED_VALUE);
```

Comment:
The repository name can't be null ever. So, the UNDEFINED_VALUE field is not necessary.